Assets/Scripts/SS3D/Systems/Input/Controls.inputactions

```diff
@@ -0,0 +1,587 @@
+{
```

Comment:
I'm going to need help with understanding this one too.

Reply:
This is where the input settings are. You can change control maps/action maps/actions and their bindings/interactions/processors through the editor